store/logger.go

```diff
@@ -88,11 +89,12 @@ func (tr *LoggedTransport) getFlags() debugflag {
 
 // returns a new http.Client with a LoggedTransport, a Timeout and preservation
 // of range requests accross redirects
-func newHTTPClient(timeout time.Duration) *http.Client {
+func newHTTPClient(timeout time.Duration, mayLogBody bool) *http.Client {
```

Comment:
I wonder if we could have a  `struct httpClientOpts{timeout time.Duration, mayLogBody bool} ` to make it eaiser to read the bool?